### PR TITLE
Create stable branch and remove unstable parts.

### PR DESCRIPTION
**Pull request by uhofemei**
Signed-off-by: Ulf Hofemeier <ulf.hofemeier@intel.com>

### CODE REVIEW

**Comment by uhofemei** [comment]
I pushed a more conclusive commit message and updated the ocf-server README file. As I don't know the environmental sensor as part of my demo kit, I can't tell whether this component is stable or not. @kiendinh, what do you think?

**Comment by kiendinh** [comment]
@uhofemei, the instability being talked about involves iotivity-constrained and this environmental-sensor is not dependent on.

**Comment by uhofemei** [comment]
So like the DC power meter, which isn't using it either. I will restore the environmental sensor code in the stable branch.

**Comment by uhofemei** [comment]
The environmental sensor code has been restored in this PR.

**Comment by gvancuts** [comment]
Can you also fix the ocf-servers/README.md file as requested? Tx

**Comment by uhofemei** [comment]
@gvancuts, done. https://github.com/01org/SmartHome-Demo/pull/168/commits/49c81fc863dcd46d6538f746ac0a6e94297b4875

File: ocf-servers/README.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 # The Smart Devices (aka OCF servers)
 ## JavaScript OCF servers
-A number of JavaScript-based OCF servers are provided in the [`js-servers`](./js-servers/) and [`zjs-servers`](./zjs-servers/) folders for `node.js` and [`zephyr.js`](https://github.com/01org/zephyr.js) respectively. The `node.js` dependencies (that can be installed using `npm`) are:
+A number of JavaScript-based OCF servers are provided in the [`js-servers`](./js-servers/)
+folder for `node.js`. The `node.js` dependencies (that can be installed using `npm`) are:
 * [IoTivity-node](https://www.npmjs.com/package/iotivity-node)
 * [mraa](https://www.npmjs.com/package/mraa)
-* [noble](https://www.npmjs.com/package/noble) (only for some)
+* [noble](https://www.:wqnpmjs.com/package/noble) (only for some)
```
Comment (gvancuts) [inline ocf-servers/README.md]: Were you trying to use `vim` when editing the file by any chance ;-)

The extra `:wq` breaks the URL, can you fix it?

Comment (uhofemei) [inline ocf-servers/README.md]: Dang, Geoffroy. You have eyes like a hawk. Good catch! I'm a vim user and I removed the failed attempt to save the file :-)

**Comment by gvancuts** [comment]
The single commit seems to include an extra previous commit (that was signed-off by me)... and the same commit message is repeated twice and the extra `:wq` is still there.. :(

**Comment by uhofemei** [comment]
How do I fix this now? Looks like the squashing ended up being on top of your last commit. Every attempt to push a fix makes it worse.

**Pull request closed without merging**